src/base/QXmppDiscoveryIq.cpp

```diff
@@ -220,7 +220,7 @@ QByteArray QXmppDiscoveryIq::verificationString() const
 
             QStringList keys = fieldMap.keys();
             std::sort(keys.begin(), keys.end());
```

Comment:
While we're at this, I'm not sure `keys` need to be sorted. The joy of tree-based `QMap` is that it naturally stores them in order.

Reply:
I dont want to touch the discoveryiq too much in this pr.